File: 2942-find-words-containing-character/2942-find-words-containing-character.java
class Solution {
    public List<Integer> findWordsContaining(String[] words, char x) {
        ArrayList<Integer> list = new ArrayList<>();
        for(int i=0;i<words.length;i++){
            for(int j=0;j<words[i].length();j++){
                if(words[i].charAt(j) == x){
                    list.add(i);
                    break;
                }
            }
        }
        return list;
    }
}

File: 2942-find-words-containing-character/2942-find-words-containing-character.js
/**
 * @param {string[]} words
 * @param {character} x
 * @return {number[]}
 */
var findWordsContaining = function(words, x) {
    let arr = [];
    for(let j =0;j<words.length;j++){
        for(let i = 0;i<words[j].length;i++){
            if(words[j].charAt(i) === x){
                arr.push(j);
                break;
            }
        }
    }
    return arr;
};